org.civicrm.bootstrapcivihr/scss/overrides/crm/_spinner.scss

```diff
@@ -0,0 +1,13 @@
+/**
+ * Overriding the default civicrm core spinner
+ * defined in file sites/all/modules/civicrm/css/civicrm.css
+ */
+.crm-container {
+  .dataTables_processing,
+  .blockUI.blockOverlay {
+    @include spinner();
+    height: 32px !important;
```

Comment:
that's not what i was talking about, and it's wrong cause what's the point of providing the `$size` property in the mixin if you then redefine width and height outside of it. Use a more specific/heavier selector so that you don't need to use `!important` in the css